macOSUpgrade.sh

```diff
@@ -191,6 +192,9 @@ declare -a startosinstallOptions=()
 ## Determine binary name
 binaryNameForOSInstallerSetup=$([ "$installerVersionMajor" -ge 11 ] && /bin/echo "osinstallersetupd" || /bin/echo "osinstallersetupplaind")
 
+## Determine binary for checksum
+binaryNameForCheckSum=$([ "$installerVersionMajor" -ge 11 ] && /bin/echo "InstallESD" || /bin/echo "SharedSupport")
```

Comment:
`$installerVersionMajor` is the middle number of version. like `10.${installerVersionMajor}.${installerVersionMinor}`
So, we need a new parameter for BigSur.

```
installerVersionNumber="$( /bin/echo "$installerVersion" | /usr/bin/awk -F. 'N = $1 * 10 ** 4 + $2 * 10 ** 2 + $3 {print N}' )"

macOS 10.11.3 => installerVersionNumber: 101103
macOS 11.0.1 => installerVersionNumber: 110001
```